### DISCUSSION TITLE
Is there a way to access a local object instantiated in a op via browser terminal?

### DISCUSSION THREAD

**Discussion by marcelocastrobarbosa** [cables-gl/cables_docs#237] [Answers & Questions] [answered]
if so, how to do it?

**Comment by steam0r** [comment, marked as answer]
hey,

not realls sure on what you mean, but from guessing i think you should be able to assign your object to a variable and then work along this:

https://cables.gl/docs/4_export_embed/dev_embed_vars/dev_embed_vars
https://cables.gl/docs/4_export_embed/dev_embed_functions/dev_embed_functions

hope this helps.

best,
stephan/steam
